template-manager-blueprint/src/main/resources/blueprints/configurations/logsearch_logfeeder/cloud_storage.handlebars

```diff
@@ -0,0 +1,15 @@
+{
+{{{#if-true fileSystemConfigs.cloudStorageEnabled}}}
+  "logfeeder-properties": {
+    "properties": {
+{{{#if-true sharedService.datalakeCluster}}}
+      "logfeeder.cloud.storage.mode": "hybrid"
+{{{else if-true sharedService.attachedCluster}}}
+      "logfeeder.cloud.storage.mode": "hybrid"
+{{{else}}}
+      "logfeeder.cloud.storage.mode": "cloud"
```

Comment:
What's the significance of this? Specific scenario. Standalone cluster with cloud storage will set the mode to "cloud"? What impact will that have.

Reply:
That's a good point. The standalone blueprints don't have INFRA_SOLR at the moment.
Explicitly checking logfeeder.cloud.storage.mode makes the most sense to me. That said, this is purely a function of time at this point. The dynamic check for the existence of INFRA_SOLR can happen later.

Reply:
Fixed, added checking for `INFRA_SOLR` at BP generation time.

Reply:
@sidseth Please see the updated PR description above if you are happy with the current logic. (Especially the attached WL case, where no support for a local Solr is given at the moment.)